website/src/content/deployment/runtime-configuration.mdx

```diff
@@ -63,6 +63,18 @@ An example configuration for local development:
 }
 ```
 
+Lastly, any envrionment configuration JSON file can contain placeholders for environment variables. This can be useful to avoid duplication between various `env.json` files for multiple different environments. Placeholders are specified with an expansion like syntax `${}` while additionally prefixing the environment variable name with `env:`.
+
+Imagine needing various `env.json` files which differ only in the `MC_API_URL` for three deployments of the Custom Application. We can use the `${env:MC_API_URL}` placeholder within the `env.json` to avoid needing various environment configuration files.
+
+```json
+{
+  "CDN_URL": "https://bucket.com",
+  "MC_API_URL": "${env:MC_API_URL}"
+}
+```
+
+The `MC_API_URL` placeholder can then be filled in various ways. For instance with a run script in a `package.json` as `MC_API_URL=https://mc-api.commercetools.com yarn start` or via a [dotenv](https://github.com/motdotla/dotenv)-file which is sourced before starting the application.
```

Comment:
```suggestion
The `MC_API_URL` environment variable can be provided in various ways. For example as an inline environment variable when running a script command (`MC_API_URL=https://mc-api.commercetools.com yarn start`), using a [dotenv](https://github.com/motdotla/dotenv) file or by defining the environment variables in your CI service.

<Info>

You can also pass multiple references to the same value

  key: "https://mc-api.${env:CTP_REGION}.${env.CTP_CLOUD_REGION}.commercetools.com"

</Info>
```

Reply:
Can we also keep the `<Info>` part?

Reply:
Ah missed this or it updated in the review: d57b590.